src/argilla/client/feedback/dataset.py

```diff
@@ -475,7 +475,9 @@ def iter(self, batch_size: Optional[int] = FETCHING_BATCH_SIZE) -> Iterator[List
         for i in range(0, len(self.records), batch_size):
             yield self.records[i : i + batch_size]
 
-    def push_to_argilla(self, name: Optional[str] = None, workspace: Optional[Union[str, rg.Workspace]] = None) -> None:
+    def push_to_argilla(
+        self, name: Optional[str] = None, workspace: Optional[Union[str, rg.Workspace]] = None
+    ) -> Union[str, None]:
```

Comment:
Same thing here if the change below to return the URL unless there's an exception is applied 👍🏻 

```suggestion
    ) -> str:
```